src/index.js

```diff
@@ -186,6 +231,31 @@ function canDisplayGraphiQL(request: Request, data: Object): boolean {
 }
 
 /**
+ * Helper function to produce an operation AST given a document and an
+ * operation name.
+ */
+function getOperation(documentAST: Object, operationName: ?string): ?Object {
```

Comment:
I had the same thought. I'll do that but give it some time to replace this usage to avoid bleeding-edge dependency-hell